### PR TITLE
[HL-Indy] Indy Helm 3 Upgrade

### PR DESCRIPTION
**Pull request by sauveergoel**
Signed-off-by: sauveergoel <sauveer.goel@accenture.com>

**Changelog**
- Updated Indy to helm3

**Reviewed by**
@sownak @suvajit-sarkar 

 **Linked issue**
#1026

### CODE REVIEW

File: platforms/hyperledger-indy/configuration/roles/clean/k8s_resources/tasks/main.yaml [hunk 1]
```diff
@@ -6,7 +6,7 @@
     helm_releases=($(KUBECONFIG={{ kubernetes.config_file }} helm ls -a | grep {{ organization }}- | awk '{print $1}'))
     for helm_release in "${helm_releases[@]}"
     do
-      KUBECONFIG={{ kubernetes.config_file }} helm delete ${helm_release} --purge
+      KUBECONFIG={{ kubernetes.config_file }} helm unistall ${helm_release}
```
Comment (suvajit-sarkar) [inline platforms/hyperledger-indy/configuration/roles/clean/k8s_resources/tasks/main.yaml:9]: uninstall spelling wrong

File: platforms/hyperledger-indy/charts/indy-auth-job/Chart.yaml [hunk 2]
```diff
@@ -1,5 +1,5 @@
-apiVersion: v1
-appVersion: 1
-description: A Helm chart for job indy-auth-job
+apiVersion: v2
```
Comment (suvajit-sarkar) [inline platforms/hyperledger-indy/charts/indy-auth-job/Chart.yaml:1]: here only the appVersion change to "2.0" is required; same for all charts

File: platforms/hyperledger-indy/configuration/roles/clean/k8s_resources/tasks/main.yaml (hunk 1 above)
Comment (sauveergoel) [reply platforms/hyperledger-indy/configuration/roles/clean/k8s_resources/tasks/main.yaml]: done

File: platforms/hyperledger-indy/charts/indy-auth-job/Chart.yaml (hunk 2 above)
Comment (sauveergoel) [reply platforms/hyperledger-indy/charts/indy-auth-job/Chart.yaml]: done

**Comment by suvajit-sarkar** [comment]
Looks good to me

**Pull request merged**